Dockerfile

```
@@ -4,7 +4,7 @@ FROM ubuntu:focal as app
 
 RUN apt-get update && \
     apt-get upgrade -qy && apt-get install language-pack-en locales git \
-    python3.8-dev python3-virtualenv libmysqlclient-dev libssl-dev build-essential wget unzip -qy && \
+    python3.8-dev python3-virtualenv libmysqlclient-dev libssl-dev build-essential pkg-config wget unzip -qy && \
```

It might be worth adding a comment to explain what these dependencies are for.
```suggestion
    python3.8-dev python3-virtualenv \
    # mysqlclient build dependencies
    libmysqlclient-dev libssl-dev build-essential pkg-config \
    wget unzip -qy && \
```